python/tvm/autotvm/task/nnvm_integration.py

```diff
@@ -297,13 +113,24 @@ def extract_from_multiple_graph(graphs, shapes, dtypes, target, symbols, target_
         collected tasks
     """
     import nnvm.compiler
+    import nnvm
+    import topi
 
     env = TaskExtractEnv.get()
 
+    #NOTE: To add more symbols, you only need to change the following lists
+    #nnvm symbol -> topi compute
+    SYMBOL2TOPI = {
+        nnvm.sym.conv2d: [topi.nn.conv2d, topi.nn.depthwise_conv2d_nchw,
+                          topi.nn.group_conv2d_nchw],
+        nnvm.sym.conv2d_transpose: [topi.nn.conv2d_transpose_nchw],
+        nnvm.sym.dense: [topi.nn.dense],
+    }
```

Comment:
can we combine these two `SYMBOL2TOPI` ?

Reply:
Perhaps; @merrymercy should we do this? There have also been discuss board questions from people who didn't know to add `conv2d_transpose` to the tutorial example when tuning other networks.

Reply:
The reason we don't combine them currently seems to be because the imports here are tricky to do outside of a function. Should we just delete the `extract_from_multiple_graphs` function? I do not see it used often.